dev/tests/integration/testsuite/Magento/Checkout/_files/quote_with_simple_product_saved_rollback.php

```diff
@@ -14,3 +14,5 @@
 /** @var \Magento\Quote\Model\QuoteIdMask $quoteIdMask */
 $quoteIdMask = $objectManager->create(\Magento\Quote\Model\QuoteIdMask::class);
 $quoteIdMask->delete($quote->getId());
+
+require 'simple_product_rollback.php';
```

Comment:
Great point that you find that we have fixtures without rollback
We missed to check it early
Thanks

Reply:
Thanks for the compliments. I've also added few rollbacks in other files too, can you please take a look into them as well?